lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -39,20 +41,18 @@ def setUpClass(cls):
         super(AktualizrToolsTests, cls).setUpClass()
         logger = logging.getLogger("selftest")
         logger.info('Running bitbake to build aktualizr-native tools')
-        bitbake('aktualizr-native')
+        bitbake('aktualizr-native aktualizr-device-prov')
+        bitbake('build-sysroots -c build_native_sysroot')
 
     def test_cert_provider_help(self):
         akt_native_run(self, 'aktualizr-cert-provider --help')
 
     def test_cert_provider_local_output(self):
-        logger = logging.getLogger("selftest")
-        logger.info('Running bitbake to build aktualizr-device-prov')
-        bitbake('aktualizr-device-prov')
```

Comment:
So are you expecting bitbake to have been run first? Probably okay, but causes problems if you try to run this test in isolation or in an unexpected sequence.

Reply:
I moved it to `setUpClass()`, so I guess it should always happen, no?

Reply:
Oops, yes of course. Sorry, using github on a phone is difficult.